src/js/routes/OpinionsFollowed.jsx

```diff
@@ -10,19 +9,13 @@ import OpinionsFollowedList from "../components/Organization/OpinionsFollowedLis
 import SearchBar from "../components/Search/SearchBar";
 
 export default class OpinionsFollowed extends Component {
-  static propTypes = {
-    children: PropTypes.object,
-    history: PropTypes.object,
-  };
-
   constructor (props) {
     super(props);
     this.state = {
       organizations_followed_list: [],
```

Comment:
I realize now that I did not specify this when we spoke, but we are updating all state variables to camelCase. Could you please update:
organizations_followed_list -> organizationsFollowedList